canonical-web-and-design/ubuntu.com
# Resize subscriptions

## Done

- Added a hook for resizing a contract.
- Call the resize API from the edit component and handle errors.

## QA

- Visit [/advantage?test_backend=true](https://ubuntu-com-10456.demos.haus/advantage?test_backend=true) and log in.
- Click on a subscription and then click "Edit subscription..." (you will need to find a subscription that is resizable).
- Change the number of machines and click "Resize".
- It should think for a bit and then resize.


## Issue / Card

Fixes: https://github.com/canonical-web-and-design/commercial-squad/issues/113.
Fixes: https://github.com/canonical-web-and-design/commercial-squad/issues/114.
